Package: module not found fbjs/lib/shallowEqual

[src/lib/shallowEqual.js](../blob/v0.74.2/src/lib/shallowEqual.js) is importing from `fbjs` which is not listed as a depencendy.

I am getting this error from webpack
```Module not found: Can't resolve 'fbjs/lib/shallowEqual'```

react 16.0.0
webpack 3.5.1
semantic-ui-react 0.74.2

--- a/package.json
+++ b/package.json
@@ -59,6 +59,7 @@
   "dependencies": {
     "babel-runtime": "^6.25.0",
     "classnames": "^2.2.5",
+    "fbjs": "^0.8.16",
     "lodash": "^4.17.4",
     "prop-types": "^15.5.10"
   },

